addons/mail/models/mail_channel.py

```diff
@@ -804,6 +804,11 @@ def channel_rename(self, name):
         self.write({'name': name})
         self._broadcast(self.channel_partner_ids.ids)
 
+    def channel_change_description(self, description):
+        self.ensure_one()
+        self.write({'description': description})
+        self._broadcast(self.channel_partner_ids.ids)
```

Comment:
Ideally we should send only what changed instead of everything. I know you copied `rename` but it wasn't a good example.

Reply:
@seb-odoo I don't understand what you mean ? I only send the description. Or is it related to the broadcast feature ?